.github/workflows/lint-eslint.yml

```diff
@@ -38,7 +38,7 @@ jobs:
 
     steps:
       - name: Checkout
-        uses: actions/checkout@3df4ab11eba7bda6032a0b82a6bb43b11571feac # v4.0.0
+        uses: actions/checkout@f43a0e5ff2bd294095638e18286ca9a3d1956744 # v3.6.0
```

Comment:
Any reason to downgrade the action? Dependabot updates them regularly for our workflows, so I'd rather stick to the latest version if there's no clear reason not to do so.

Reply:
uh wow... We're more up to date than the nextcloud one. Maybe we should install dependabot there. But I guess that's another can of worms :worm: 
I'll revert the downgrade and then merge ahead.